src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        final int number = 100;
+        Ball ball = new Ball(new ColorSupplier().getRandomColor(), new Random().nextInt(number));
```

Comment:
ideally, you should move creation of new objects (new ColorSupplier(), new Random()) outside the method as well (create private fields)

Reply:
Yeah, maybe, ideally i should move creation outside the method, but when i do it, i'm always have the same result:
![image](https://user-images.githubusercontent.com/111348563/217818410-cbafc493-53f3-48be-9b84-e4ae971243ed.png)
If i'd like to have a random result, like a task want, i still have to create object in method, but now in main() method. I would like to stay it creation of object in this method, cos it's doesn't metter where i will create it.